docs/samples/browser-plugin-meetings/app.js

```diff
@@ -1325,8 +1278,8 @@ function clearMediaDeviceList() {
 }
 
 function getLocalMediaSettings() {
-  if (localMedia.cameraTrack) {
-    const videoSettings = localMedia.cameraTrack.getSettings();
+  if (localMedia.cameraStream) {
+    const videoSettings = localMedia.cameraStream.getTracks()[0]?.getSettings();
```

Comment:
LocalStream class has a `getSettings()` method that calls getSettings() on the inputTrack, so we should probably use that here, right? BTW, I can't find a `getTracks()` method in the webrtc-core stream classes, so does this existing code work at all?

Reply:
I think this is a mistake. Will update it.

Reply:
Done: https://github.com/webex/webex-js-sdk/pull/3052

Reply:
Thanks @meta-dipanshu-sharma